socorro/external/postgresql/connection_context.py

```diff
@@ -6,10 +6,14 @@
 import contextlib
 import psycopg2
 import psycopg2.extensions
+import urlparse
 
 from configman.config_manager import RequiredConfig
 from configman import Namespace
 
+urlparse.uses_netloc.append('postgres')
+urlparse.uses_netloc.append('postgresql')
+
```

Comment:
These might require a comment. Why is it needed? Also, the first part of a URI is usually called the "scheme" E.g. "ftp" but this talks about "netloc" which is usually the alias for "hostname".